users/releases.rst

```diff
@@ -0,0 +1,126 @@
+Release Channels
+================
+
+There are two different release channels that can be selected. The *stable*
+channel is the more stable one, and is introduced by the change in release
+process in February, 2017. The *candidate* channel contains the usual
+bi-weekly release from the development branch.
+
+There are a number of trade-offs between the two:
+
+=========================  =========================  ======================
+\                                   Stable                   Candidate
+=========================  =========================  ======================
+**Stability**              More Stable                More Experimental
+**Features & Fixes**       About two weeks behind     Latest
+**Auto Upgrades**          Optional                   Mandatory [#]_
+**Anon. Usage Reporting**  Optional                   Mandatory
+**Support**                Fully supported            Fully supported [#]_
+=========================  =========================  ======================
+
+Run the candidate channel if you are technically savvy and enjoy new
+features. Run the stable channel if you want to minimize the amount of
+surprises you might run into.
+
+.. [#] Auto upgrades are not enabled in builds delivered via APT or Snap.
+.. [#] Yes, there is intentionally no difference here.
+
+Life Cycle
+----------
+
+Every new feature and bugfix begins its life in the development branch,
+``master``. Every two weeks the current ``master`` becomes a *release
+candidate*. This version is identified by "-rc" in it's name, for example
+"0.14.35-rc.1".
+
+Those running the candidate channel will update to this release candidate.
+During the next twelve days it receives testing "in the wild". Any new,
+serious issues that are discovered are fixed, and new release candidates
+"0.14.35-rc.2" etc are created as needed. These release candidates do not
+include any new features or non-essential bugfixes added to `master` in the
```

Comment:
Intentionally different formatting of \`master\` than in paragraph above (\`\`master\`\`)?